multifilereader.go

```diff
@@ -88,7 +89,10 @@ func (mfr *MultiFileReader) Read(buf []byte) (written int, err error) {
 			// write the boundary and headers
 			header := make(textproto.MIMEHeader)
 			filename := url.QueryEscape(path.Join(path.Join(mfr.path...), entry.Name()))
-			header.Set("Content-Disposition", fmt.Sprintf("file; filename=\"%s\"", filename))
+			name := fmt.Sprintf("data%d", mfr.fieldCounter)
```

Comment:
What do browser set this to when user picks multiple files in file input? Can one name repeat?

Reply:
https://tools.ietf.org/html/rfc7578#section-4.3

> To match widely deployed implementations, multiple files MUST be sent by supplying each file in a separate part but all with the same "name" parameter.

I suppose we want to mimic a form with multiple upload from the same field, so I should change that.